### PR TITLE
Fixing FDC3 related .Net projects to have MorganStanley.ComposeUI prefix

### PR DESCRIPTION
**Pull request by lilla28**
Changes:
- Fixed FDC3 related .Net projects to have MorganStanley.ComposeUI prefix
- Fixed FDC3 DesktopAgent's functionality when a module is started/stopped to check if the module is started by the DesktopAgent (it's having the fdc3InstanceId as Fdc3StartupProperties) instead of throwing an exception

### CODE REVIEW

File: src/fdc3/dotnet/DesktopAgent/DesktopAgent.sln [hunk 1]
```diff
@@ -5,11 +5,11 @@ VisualStudioVersion = 17.4.33103.184
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{75D43F58-806B-4C28-BDA1-7A7AFF0250DB}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DesktopAgent", "src\DesktopAgent\DesktopAgent.csproj", "{4A71A18B-6DC1-44AD-85F3-200CC2CBC6BD}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MorganStanley.ComposeUI.DesktopAgent", "src\MorganStanley.ComposeUI.DesktopAgent\MorganStanley.ComposeUI.DesktopAgent.csproj", "{4A71A18B-6DC1-44AD-85F3-200CC2CBC6BD}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{44C08E9B-2A1D-4F01-B579-59DFCE4CBF6D}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "testt", "{44C08E9B-2A1D-4F01-B579-59DFCE4CBF6D}"
```
Comment (ztanczos) [inline src/fdc3/dotnet/DesktopAgent/DesktopAgent.sln:10]: testt?

Comment (lilla28) [reply src/fdc3/dotnet/DesktopAgent/DesktopAgent.sln:10]: that happens when i stop wearing my glasses, I will fix that :)

File: src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/Fdc3DesktopAgent.cs [hunk 2]
```diff
@@ -34,6 +34,7 @@
 using Screenshot = MorganStanley.ComposeUI.Fdc3.DesktopAgent.Protocol.Screenshot;
 using MorganStanley.ComposeUI.Fdc3.DesktopAgent.Protocol;
 using System.Collections.Generic;
+using System.Reflection.Metadata.Ecma335;
```
Comment (bingenito) [inline src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/Fdc3DesktopAgent.cs:37]: Should this be part of this PR?

File: src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/Fdc3DesktopAgent.cs
Comment (ZKRobi) [inline src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/Fdc3DesktopAgent.cs:1]: Are the changes in this file intentional?

Comment (lilla28) [reply src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/Fdc3DesktopAgent.cs:1]: I am rebasing the PR and separate the commit into 2.

**Review by ZKRobi — APPROVED** [review]

**Pull request merged**